src/target/lpc546xx.c

```diff
@@ -182,20 +180,13 @@ static void lpc546xx_reset_attach(target_s *t)
 	cortexm_attach(t);
 }
 
-static bool lpc546xx_mass_erase(target_s *t)
-{
-	const int result = lpc546xx_flash_erase(t->flash, t->flash->start, t->flash->length);
-	if (result != 0)
-		tc_printf(t, "Error erasing flash: %d\n", result);
-	return result == 0;
-}
-
 static bool lpc546xx_cmd_erase_sector(target_s *t, int argc, const char **argv)
 {
+	/* FIXME: is this command really needed? feels like it should at best be a generic target command */
```

Comment:
It is probably entirely superseded by the `mon erase_range` command, which is considerably more flexible

Reply:
Hm, I think this was a note to myself to investigate further, shouldn't have made it through to the PR

Any actions to take here?

Reply:
Think having it display a message saying to use `mon erase_range` instead, marking it deprecated, would be a good choice - that then opens up the possibility of removing it in a next release. Possibly included in this behaviour change would be having it call the erase range machinery or making the command a no-op.

Reply:
Added a deprecation message, did not change anything with the actual command because it boils down to a `target_flash_erase` api call, which is in essence the same as erase_range already